src/VirtualColumn.php

```diff
@@ -13,74 +16,113 @@
  */
 trait VirtualColumn
 {
-    public static $afterListeners = [];
+    public static array $customEncryptedCastables = [];
```

Comment:
Add a docblock to make it clear what this is used for

Reply:
```suggestion
    /**
     * Encrypted castables have to be decoded using a special approach
     * that prevents the data from getting encrypted repeatedly (in decodeVirtualColumn()).
     *
     * The default encrypted castables ('encrypted', 'encrypted:array', 'encrypted:collection', 'encrypted:json', 'encrypted:object'),
     * are already handled, so you only have to add your custom encrypted castables here.
     */
    public static array $customEncryptedCastables = [];
```